lib/guardian.rb

```diff
@@ -445,23 +445,39 @@ def can_invite_group_to_private_message?(group, topic)
     can_send_private_message?(group)
   end
 
-  def can_send_private_message?(target, notify_moderators: false)
-    is_user = target.is_a?(User)
-    is_group = target.is_a?(Group)
+  ##
+  # This should be used as a general, but not definitive, check for whether
+  # the user can send private messages _generally_, which is mostly useful
+  # for changing the UI.
```

Comment:
This suggestion is a bit silly, but is it possible to also indicate there's an overloaded method directly below, in case the reader misses it? e.g.

> Please otherwise use `can_send_private_message?(target, notify_moderators)` to check if the target can be messaged.

Reply:
Not silly, makes sense! Will add :)